modules/lazy-images.php

```diff
@@ -2,7 +2,7 @@
 
 /**
  * Module Name: Lazy Images
- * Module Description: Improve performance by loading images just before they scroll into view
+ * Module Description: "Lazy load" images
```

Comment:
IMHO the previous description was much more descriptive for non tech savvy users.